packages/next/test/fixtures/00-app-dir-edge-404/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "dependencies": {
     "next": "canary",
-    "react": "experimental",
-    "react-dom": "experimental"
+    "react": "19.0.0-beta-4508873393-20240430",
+    "react-dom": "19.0.0-beta-4508873393-20240430"
```

Comment:
I guess these can stay `experimental` since they never matched the peer dependency range to begin with?

Reply:
Was using the tag bypassing the peer dep check maybe?

Reply:
It shouldn't because the peer dep check is done on the resolved version not on the tag.